samples/nrf54h20/idle_relocated_tcm/remote/src/main.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(idle);
+
+#define CODE_PARTITION_NODE		DT_CHOSEN(zephyr_code_partition)
+#define CODE_PARTITION_START_ADDR	DT_FIXED_PARTITION_ADDR(CODE_PARTITION_NODE)
+
+int main(void)
+{
+	unsigned int cnt = 0;
+	uintptr_t pc;
+
+	__asm__ volatile("adr %0, ." : "=r"(pc));
+	LOG_INF("Multicore idle test on %s", CONFIG_BOARD_TARGET);
+	LOG_INF("Original code partition start address: 0x%lx",
+		(unsigned long)CODE_PARTITION_START_ADDR);
+	LOG_INF("Current PC (program counter) address: 0x%lx", (unsigned long)pc);
```

Comment:
should get the proper start location from linker or dts/kconfig

Reply:
well none of the application will run if it's not running from the exact address it was built for, so an image built for MRAM will not from from TCM. The reason to use the start address of the image is so you can test this, as it stands right now you can't make a test for this because with each build or kernel change or compiler change or zephyr change, the address is going to change